examples/python/gRPC/meta/gRPC_fb_loadProjects.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+
+import flatbuffers
+from seerep.fb.Empty import Empty
+from seerep.fb.meta_operations_grpc_fb import MetaOperationsStub
+from seerep.util.common import get_gRPC_channel
+from seerep.util.fb_helper import createEmpty
+
+channel = get_gRPC_channel()
+
+stub = MetaOperationsStub(channel)
+
+fbb = flatbuffers.Builder(1024)
+responseBuf = stub.LoadProjects(bytes(createEmpty(fbb)))
+responseBuf = Empty.GetRootAs(responseBuf)
```

Comment:
This should be ProjectInfos, shouldn't it?

Reply:
Yes, I will also add printing the received UUIDs